src/SprykerEco/Service/ComputopApi/ComputopApiServiceFactory.php

```diff
@@ -46,21 +45,10 @@ public function createComputopApiMapper(): ComputopApiMapperInterface
     {
         return new ComputopApiMapper(
             $this->getConfig(),
-            $this->createTextService()
+            $this->getProvidedDependency(ComputopApiDependencyProvider::SERVICE_UTIL_TEXT),
```

Comment:
Please revert, we should have a separate method for this

Reply:
Fixed